src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceIntTest_getOffenderAlerts.java

```diff
@@ -208,7 +208,7 @@ void shouldHaveTheCorrectRoleToAccessEndpoint() {
         @DisplayName("returns partial alert data for each alert for each booking")
         void returnsImportantAlertDataForEachAlertForEachBooking() {
             final var response = testRestTemplate.exchange(
-                "/api/offenders/{offenderNo}/alerts/v2",
+                "/api/offenders/{offenderNo}/alerts/v2?sort=alertType,alertId,dateExpires",
```

Comment:
default sort is alertType which doesn't help when there are multiple results with the same alertType. I've therefore added a few more fields to get the same sort order as previously.